tools/docs-fetcher.py

```diff
@@ -69,6 +69,14 @@ def fetch_docs(file_path):
         docs['repo_name'] = repo_name
         docs['file'] = file_path
 
+        if docs.get('buildDocsCmd'):
+            print(
+                "Running build command: ",
+                docs.get('buildDocsCmd'),
+                os.path.join(TOP_DIR_PATH, EXTERNAL_REPOS_DIR, repo_name)
+            )
+            subprocess.run(docs.get('buildDocsCmd'), shell=True, cwd=os.path.join(TOP_DIR_PATH, EXTERNAL_REPOS_DIR, repo_name))
```

Comment:
I prefer not having to repeat the build key 3 times, so we should use a var for this. Same for the running path.